### PR TITLE
chore(ci): push-ci updates

### PR DESCRIPTION
**Pull request by mwangggg**
# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
* [x] Read the [contributing guidelines](https://github.com/cryostatio/cryostat/blob/main/CONTRIBUTING.md)
* [x] Linked a relevant issue which this PR resolves
* [x] Linked any other relevant issues, PR's, or documentation, if any
* [x] Resolved all conflicts, if any
* [x] Rebased your branch PR on top of the latest upstream `main` branch
* [x] Attached at least one of the following labels to the PR: `[chore, ci, docs, feat, fix, test]`
* [x] [Signed all commits using a GPG signature](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification#gpg-commit-signature-verification)

**To recreate commits with GPG signature** `git fetch upstream && git rebase --force --gpg-sign upstream/main`
_______________________________________________

Related to: #1783

### CODE REVIEW

File: .github/workflows/push-ci.yml [hunk 1]
```diff
@@ -15,58 +15,48 @@ on:
...
+          podman load -i cryostat-${{ matrix.arch }}.tar
+          podman tag quay.io/cryostat/cryostat:latest $CRYOSTAT_IMG:$IMAGE_VERSION-linux-${{ matrix.arch }}
     - name: Create OCI Manifest
       id: create-manifest
       env:
-        IMAGE_VERSION: ${{ needs.build-and-test-amd64.outputs.image-version }}
+        IMAGE_VERSION: ${{ needs.build-and-test.outputs.image-version }}
       run: |
-        podman manifest create $CRYOSTAT_IMG:$IMAGE_VERSION containers-storage:$CRYOSTAT_IMG:$IMAGE_VERSION-linux-amd64 containers-storage:$CRYOSTAT_IMG:$IMAGE_VERSION-linux-arm64
+        podman manifest create $CRYOSTAT_IMG:$IMAGE_VERSION containers-storage:$CRYOSTAT_IMG:$IMAGE_VERSION-linux-${{ matrix.arch }}
```
Comment (andrewazores) [inline .github/workflows/push-ci.yml:59]: I'm not sure this is going to work as expected - I think the matrix setup might make this `podman manifest create` command run two separate times (once for each arch value), but it really needs to run once with all of the arches in a list like it was before.

**Comment by andrewazores** [comment]
Any more changes planned?

**Comment by mwangggg** [comment]
> Any more changes planned?

nope. Sorry I didn't get this one in before the last PR closed haha

**Review by andrewazores — APPROVED** [review]

**Pull request merged**